whois-api/src/main/java/net/ripe/db/whois/api/rdap/AutnumException.java

```diff
@@ -0,0 +1,28 @@
+package net.ripe.db.whois.api.rdap;
+
+public class AutnumException extends RuntimeException {
```

Comment:
Can you make `AutnumException` extend `RdapException` ?

Reply:
I was thinking of doing that but the subclass will be the same as the super class, so it will just extend RdapException but it will be empty.
I will do that because for me it is more clear (AutnumException is an RdapException, they are related)